### PR TITLE
x11-libs/motif: use HTTPS, add missing die

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

Simple update to the ebuild to use https and add some missing die's.
Please review.

Signed-off-by: Michael Mair-Keimberger <m.mairkeimberger@gmail.com>

### CODE REVIEW

**Comment by ulm** [comment]
Sorry, but merging

File: x11-libs/motif/motif-2.3.8-r1.ebuild [hunk 1]
```diff
@@ -108,7 +108,7 @@ multilib_src_install_all() {
 	newins "${FILESDIR}"/Mwm.defaults Mwm
 
 	# cleanup
-	rm -rf "${ED}"/usr/share/Xm
+	rm -rf "${ED}"/usr/share/Xm || die
```
Comment (ulm) [inline x11-libs/motif/motif-2.3.8-r1.ebuild]: rm -f won't error out, even if a file isn't found, so the die is pretty much useless here.

Comment (mm1ke) [reply x11-libs/motif/motif-2.3.8-r1.ebuild]: Updated

**Comment by ulm** [comment]
I've updated the HOMEPAGE to use https.

Otherwise, NACK on changing the stable ebuild in place, and I think it's a tiny issue that doesn't justify a revbump either. I have opened [bug 677488](https://bugs.gentoo.org/677488) for this (so that it won't be forgotten).

File: x11-libs/motif/motif-2.3.8-r1.ebuild [hunk 2]
```diff
@@ -39,10 +39,10 @@ src_prepare() {
 	eapply_user
 
 	# disable compilation of demo binaries
-	sed -i -e '/^SUBDIRS/{:x;/\\$/{N;bx;};s/[ \t\n\\]*demos//;}' Makefile.am
+	sed -i -e '/^SUBDIRS/{:x;/\\$/{N;bx;};s/[ \t\n\\]*demos//;}' Makefile.am || die
```
Comment (ulm) [inline x11-libs/motif/motif-2.3.8-r1.ebuild]: Long line (more than 80 chars).

**Pull request closed without merging**